lib/ain-ocean/src/model/governance.rs

```diff
@@ -0,0 +1,89 @@
+use defichain_rpc::json::governance::{ProposalInfo, ProposalStatus, ProposalType};
+use serde::{Deserialize, Serialize, Serializer};
+use serde_with::skip_serializing_none;
+
+#[skip_serializing_none]
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ApiProposalInfo {
+    pub proposal_id: String,
+    pub title: String,
+    pub context: String,
+    pub context_hash: String,
+    pub r#type: ProposalType,
+    pub status: ProposalStatus,
+    #[serde(
+        serialize_with = "serialize_amount",
+        skip_serializing_if = "should_skip"
+    )]
```

Comment:
we don't need this.. return null will do

Reply:
if I dont use `should_skip `, the amount params does not appear, so we need have `should_skip `.

Reply:
fixed [a286aa1](https://github.com/DeFiCh/ain/pull/2947/commits/a286aa1abbbe83ee58c34792cc4f5d16a654e78a)

Reply:
@nagarajm22 my fixed got erased......

Reply:
I took the latest and did changes in above serialisation :) can you undo it ?

Reply:
fixed [b7879e7](https://github.com/DeFiCh/ain/pull/2947/commits/b7879e717a42b2069ad0c7ea9230d2c5c3199a83)